airbyte-db/src/main/resources/schema.sql

```diff
@@ -8,7 +8,8 @@
  CREATE
     EXTENSION IF NOT EXISTS "uuid-ossp";
 
--- types
+-- Statements Below this line are being re-used to create new schemas
```

Comment:
these tags are useful to separate the commands that can be run in the java process vs the docker process when init database